ext/styledmap/eu.esdihumboldt.hale.doc.user.views.styledmap/html/customtile_detail.html

```diff
@@ -0,0 +1,103 @@
+		using Open Perspective tool menu reside in the right corner of the studio.
+    </p>
+    
+    <div>
+		<img src="../images/map_perspective_toolbar.png"
+			title="Map perspective tool menu" class="fit">
+		<img src="../images/map_perspective_open.png"
+			title="Map perspective tool menu" class="fit" style="margin-left:20px" >
+	</div>
+    
+    <p>
+		Custom tile map is a default map in <i>HALE</i>. By default you will see,
+		<i>Stamen Terrain</i> map [Map tiles by Stamen Design, under CC BY 3.0. 
+		Data by OpenStreetMap, under CC BY SA.]
+    </p>
+    
+    <div>
+		<img src="../images/custom_default.png"
+			title="Custom Tile map(Stamen Terrain) as a default in HALE" class="fit">
+		<img src="../images/custom_default_menu.png"
+			title="Stamen Terrain in view menu of map" class="fit" style="margin-left:20px" >
+	</div>
+ 
+    
+    <p>
+    	To add the <i>Custom Tile</i> maps in to Map perspective, click on the <i>View Menu</i> 
+    	button and then heads to <i>Custom Tile Maps</i>&#8594;<i>Add</i>. 
+	</p>
+	
+	<div>
+		<img src="../images/custom_add.png"
+			title="Add Custom Tile map to Map perspective" class="fit">
+	</div>
+	
+	
+	<p>
+		By clicking on <i>Add</i> button, a dialog will be appeared, which asks 
+		for the <i>Name</i> and <i>URL Pattern</i>. You have to insert a unique 
+		name and URL pattern from where tiles will be fetched. 
+	</p>
+	<p>
+		Please note, here name must be unique from any other <i>Custom Tile</i> 
+		map configurations and URL pattern must include {z}, {x} and {y} 
+		literals which will be replaced by the HALE as Zoom level, x-coordinate 
+		and y-coordinate respectively while fetching the tiles. To go to the 
+		next configuration page, click on <i>Next</i> button in the dialog or 
+		click <i>Finish</i> directly to skip next configuration pages.
+	</p>
+	<div>
+		<img src="../images/custom_basic.png"
+			title="Basic configuration of Custom Tile maps" class="fit">
+	</div>
+	
+	<p>
+		On this page, You have to define the maximum <i>zoom level</i> for the 
+		map (default is 16) and attribution text of the map server.	Click 
+		<i>Finish</i> to complete the configuration.
+		
+		Please note, before using a map you should make sure that you doing so 
+		is in compliance with the respective terms of use.
+	</p>
+	<div>
+		<img src="../images/custom_extension.png"
+			title="Zoom level and Attribuition configuration of Custom Tile maps" class="fit">
+	</div>
+	
+	
+	<p>
+		After finishing configuration, <i>Custom Tile maps</i> will be loaded to <i>Map</i>
+		perspective and also added in to the <i>view menu</i>. 
+	</p>
+	<div>
+		<img src="../images/custom_view.png"
+			title="Custom Tile map loaded in to Map perspective" class="fit">
+	</div>
+	
+	<p>
+		You can again reconfigure the map added by you, by clicking on the name you
+		configured before. It will reopen the same dialogs as per above in same order.
+		You can change the configurations in those dialogs. 
```

Comment:
Replace paragraph with: 
"To change the configuration for a custom tile map, click on the name you set before. Change any of the settings by stepping through the dialog again."